### ISSUE TITLE
Uncaught TypeError: boolean is not a function

### ISSUE THREAD

**Issue by intellix** [AtomLinter/linter-eslint#90]
[Enter steps to reproduce below:]

1. Cmd+P and entered a file: "app.js"
2. Error popped up prompting me to submit this report.

**Atom Version**: 0.206.0
**System**: Mac OS X 10.10.3
**Thrown From**: [linter-eslint](https://github.com/AtomLinter/linter-eslint) package, v0.5.8

### Stack Trace

Uncaught TypeError: boolean is not a function

```
At /Users/dominic/Sites/el-desktop/node_modules/eslint/node_modules/object-assign/index.js:19

TypeError: boolean is not a function
  at /Users/dominic/Sites/el-desktop/node_modules/eslint/node_modules/object-assign/index.js:19:14
  at Array.filter (native)
  at ownEnumerableKeys (/Users/dominic/Sites/el-desktop/node_modules/eslint/node_modules/object-assign/index.js:18:14)
  at module.exports (/Users/dominic/Sites/el-desktop/node_modules/eslint/node_modules/object-assign/index.js:30:10)
  at addDeclaredGlobals (/Users/dominic/Sites/el-desktop/node_modules/eslint/lib/eslint.js:139:5)
  at EventEmitter.module.exports.api.verify (/Users/dominic/Sites/el-desktop/node_modules/eslint/lib/eslint.js:702:13)
  at /Users/dominic/.atom/packages/linter-eslint/lib/linter-eslint.coffee:129:25
  at exports.allowUnsafeNewFunction (/Users/dominic/.atom/packages/linter-eslint/node_modules/loophole/lib/loophole.js:25:14)
  at LinterESLint.lintFile (/Users/dominic/.atom/packages/linter-eslint/lib/linter-eslint.coffee:128:7)
  at /Users/dominic/.atom/packages/linter/lib/linter-view.coffee:163:18
  at Array.forEach (native)
  at /Users/dominic/.atom/packages/linter/lib/linter-view.coffee:162:18
  at FSReqWrap.oncomplete (fs.js:77:15)

```

### Commands

```
     -8:14.5.0 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused)
     -8:13.5.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -8:13.5.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -7:53.1.0 core:undo (atom-text-editor.editor.is-focused)
     -7:52.2.0 core:copy (atom-text-editor.editor.is-focused)
     -7:51.8.0 core:save (atom-text-editor.editor.is-focused)
     -7:43.7.0 core:undo (atom-text-editor.editor.is-focused)
  2x -7:43.1.0 core:redo (atom-text-editor.editor.is-focused)
  5x -7:42 editor:select-to-beginning-of-word (atom-text-editor.editor.is-focused)
     -7:40.9.0 core:copy (atom-text-editor.editor.is-focused)
     -7:38.3.0 core:undo (atom-text-editor.editor.is-focused)
     -7:37.7.0 core:save (atom-text-editor.editor.is-focused)
     -0:13.9.0 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused)
     -0:13.8.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -0:12.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:02.8.0 core:copy (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "monokai"
    ],
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ]
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.3.0
atom-jasmine, v0.5.0
color-picker, v2.0.4
editor-stats, v0.17.0
emmet, v2.3.10
highlight-selected, v0.9.3
linter, v0.12.6
linter-eslint, v0.5.8
minimap, v4.9.4
monokai, v0.14.0
project-manager, v1.15.8

# Dev
No dev packages
```

**Comment by iam4x** [comment]
Issue with `eslint` not the package, they should be catched now.
